paperscraper/utils.py

```diff
@@ -104,22 +98,20 @@ async def _allow(self) -> None:
 
     async def _request(self, *args, **kwargs) -> aiohttp.ClientResponse:
         """Throttled _request()."""
-        for retries in range(5):
-            await self._allow()
+        for retry_num in range(5):
```

Comment:
I think that retry num max should be exposed as a `ThrottledClientSession` attribute which defaults to 5. I could see wanting this to fail fast at 1.